node/src/BaseClient.ts

```diff
@@ -7729,4 +7730,12 @@ export class BaseClient {
 
         return response;
     }
+    /**
+     * Return a statistics
+     *
+     * @return Return an object that contains the statistics collected internally by GLIDE core
+     */
+    public async getStatistics(): Promise<object> {
```

Comment:
Why do we need the `Promise`? we can return the `object` directly. This method does not involve I/O it returns immediately